src/pages/NoticeEmployerPage/loader/noticeEmployerLoader.ts

```diff
@@ -0,0 +1,25 @@
+import { LoaderFunction } from "react-router-dom";
+
+import { loadNotice, loadRecentNotices } from "@/apis/loaders/notice";
+import { useUserStore } from "@/hooks/useUserStore";
+
+const noticeEmployerLoader: LoaderFunction = async ({ params }) => {
+  const user = useUserStore.getState().user;
+  const { shopId, noticeId } = params as {
+    shopId: string;
+    noticeId: string;
+  };
+
+  if (user?.shopId === shopId) {
+    const [noticeInfo] = await Promise.all([loadNotice({ shopId, noticeId })]);
+
+    return { noticeInfo };
+  }
+
+  const [noticeInfo] = await Promise.all([loadNotice({ shopId, noticeId })]);
```

Comment:
noticeInfo를 중복해서 불러올 필요 없을 것 같습니다!

```suggestion
const noticeInfo = await loadNotice({ shopId, noticeId });
if (user?.shopId === shopId) return { noticeInfo };

const recentNotices = await loadRecentNotices(noticeId);
return { noticeInfo, recentNotices };

```